src/ansys/dynamicreporting/core/utils/report_remote_server.py

```diff
@@ -37,6 +37,16 @@
 from .encoders import BaseEncoder
 
 
+class TemplateEditorJSONLoadingError(Exception):
```

Comment:
I created a customized exception class to specially representing the JSON loading error.